src/app/landing/LandingPage.css

```diff
@@ -249,3 +254,26 @@
     font-size: 11px;
   }
 }
+
+@media screen and (max-width: 1224px) {
+  .LandingPage .IntroBody {
+    background: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.6)),url('/img/landing_page/top1_mobile.jpg') no-repeat center center fixed;
+    background-position: 30% 0%;
+    background-size: cover;
+  }
+}
+
+@media screen and (min-width: 1224px) {
+  .IntroTop, .IntroBottom, .OnboardingText {
+    max-width: 800px;
+  }
+
+  .OnboardingText {
```

Comment:
Not really sure why OnboardingText needs its own section, considering just putting it in the one below would already give it the margins and max width, but alright.

Reply:
Fixed!